static/js/settings_display.js

```diff
@@ -84,16 +102,20 @@ exports.set_up = function () {
         } else {
             context.enabled_or_disabled = i18n.t('Disabled');
         }
+        var spinner = $("#display-settings-status").expectOne();
+        loading.make_indicator(spinner, {text: 'Saving ...'});
+        exports.set_margin();
 
         channel.patch({
             url: '/json/settings/display',
             data: data,
             success: function () {
-                ui_report.success(i18n.t("High contrast mode __enabled_or_disabled__!", context),
+                ui_report.success(i18n.t(" Saved."),
```

Comment:
extra space here before Saved?

Reply:
This is because I thought we need a space between checkmark and the other texts.